package.json

```diff
@@ -6,14 +6,16 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint --fix"
+    "lint": "next lint --fix",
```

Comment:
Currently, `npm run lint` automatically fixes errors (but not committing) instead of reporting them.

To make the hook useful, we might want to notify developers when errors are found so they can be addressed before pushing.

We can achieve this by creating a separate script that runs linting without auto-fixing:
```json
"lint": "next lint",
"lint:fix": "next lint --fix"
```